src/test/java/misc/ParserTest.java

```diff
@@ -0,0 +1,28 @@
+package misc;
+

```

Comment:
You might want to rearrange the import statements as they are in the wrong order.